src/metabase/driver/sql/query_processor.clj

```diff
@@ -559,8 +640,8 @@
            (->float driver numerator)
            (for [denominator denominators]
              (hx/call :case
-               (hx/call := denominator 0) nil
-               :else                        denominator)))))
+                      (hx/call := denominator 0) nil
```

Comment:
Above (line 303), `(hx/call :=...` was unwrapped to `[:=` I don't know if we want to be consistent in this ns.

Reply:
hmmm. We probably should have left this as `(hx/call := ...)` actually. Sometimes `[:= x y]` is ok but sometimes it confuses Honey SQL 1, e.g. if you use it in a `:select` without an alias. I guess we never actually emit columns in `SELECT` with no alias which is why this didn't break anything. I'll change it back